info.csv

```diff
@@ -16,4 +16,4 @@ Ajay pratap singh,       17scse101198,           1713101188,            Android,
 rahul,                   17scse112017,           1713112001,            vui,                            2,            1-DA,     javascript,                   vui AND visualizations,     NA,                 rahul.chauhan049@gmail.com,       https://github.com/rahulchauhan049,   https://www.linkedin.com/in/rahul-chauhan-818481b7/,      https://twitter.com/calledRahul
 Himanshu Singh,          17SCSE101209,           1713101199,            None,                            2nd,         6,        C++,                         "Java,Web Development",  https://lazycipher.github.io/#projects, imhsingh16@gmail.com, https://github.com/lazycipher/,     https://www.linkedin.com/in/himanshu-singh16/ ,          Manners Maketh Man
 PRIYANSHU TRIVEDI,       17SCSE105051,           1713105051,            NONE,                            2nd,         1-cloud,  C C++ PYTHON,                 competitive coding,        NA,                  trivedipriyanshu972@gmail.com,    https://github.com/PRIYANSHU-AMBITION,   https://www.linkedin.com/in/priyanshu-trivedi-54020b189/,  NA
-DANISH JAMAL,            18SCSE1180046,          18021180045,           ANDROID DEV,                    1,            7-ML & Ai C, PYTHON, JAVA               ANDROID, ML&AI, DJANGO     MIWOK, CACULATOR     danishjamal.104@gmail.com         https://github.com/danishjamal104        https://www.linkedin.com/in/danish-jamal-87b59517b        Every 1 must know how to code....          
+DANISH JAMAL,            18SCSE1180046,          18021180045,           ANDROID DEV,                    1,            7-ML & Ai C, PYTHON, JAVA,               ANDROID, ML&AI, DJANGO,     MIWOK, CACULATOR,     danishjamal.104@gmail.com,         https://github.com/danishjamal104,        https://www.linkedin.com/in/danish-jamal-87b59517b,        Every 1 must know how to code....          
```

Comment:
put all the data in " " if you have multi-value in any columns 
like in area of expertise is C, Python, java should be like:-  "C, Python, java",